.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,6 +7,6 @@
 [List any existing issues this PR resolves]
 
 ### Check List
-- [ ] All tests pass. See https://github.com/sous-chefs/apache2/blob/master/TESTING.md
+- [ ] All tests pass. See https://github.com/sous-chefs/mariadb/blob/master/TESTING.md
```

Comment:
Include the file TESTING.md in the PR, as it does not exist on mariadb (but exist on apache2)

Reply:
done 👍